src/joint_trajectory_action/joint_trajectory_action.py

```diff
@@ -354,6 +354,19 @@ def _on_trajectory_action(self, goal):
         control_rate = rospy.Rate(self._control_rate)
 
         dimensions_dict = self._determine_dimensions(trajectory_points)
+
+        if num_points == 1:
+            # Add current position as trajectory point
+            first_trajectory_point = JointTrajectoryPoint()
+            first_trajectory_point.positions = self._get_current_position(joint_names)
+            if dimensions_dict['velocities']:
```

Comment:
Very interesting. This is an edge case to be sure, but we could copy out the one trajectory point's velocities and accelerations and place them in this crafted start point. This would allow us to preserve our force-to-zero accelerations / velocities for the final (only) trajectory point.